client-src/elements/chromedash-metadata.ts

```diff
@@ -1,39 +1,33 @@
 import {LitElement, css, html} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
 import {ifDefined} from 'lit/directives/if-defined.js';
 import {SHARED_STYLES} from '../css/shared-css.js';
 
+@customElement('chromedash-metadata')
 class ChromedashMetadata extends LitElement {
```

Comment:
It looks like this element is fetched from other files using `document.querySelector('chromedash-metadata')`, so check out https://lit.dev/docs/components/defining/#typescript-typings for how to give those calls the right types.